index.js

```diff
@@ -0,0 +1,78 @@
+//DOM Variables 
+let resultContainer = document.querySelectorAll("article div")[5];
+let errorContainer = document.querySelectorAll("article div")[2];
+let calculateButton = document.getElementsByClassName('btn btn-danger')[0];
+let firstOperand = document.getElementById("firstOperand");
+let secondOperand = document.getElementById("secondOperand");
+let thirdOperand = document.getElementById("thirdOperand");
+let fourthOperand =  document.getElementById("fourthOperand");
+
+resultContainer.innerHTML = '';
+errorContainer.innerHTML = '';
+
+// Get the operation sign   --- >>  (TO DO: A FOR LOOP or something to respect DRY !!!!!!!!!)
+let op = document.getElementsByName("operations");
+var operationSign1 = op[0].options[op[0].selectedIndex].text;
+op[0].addEventListener('change', function(event) {
+    operationSign1 = op[0].options[op[0].selectedIndex].text;
+});
+var operationSign2 = op[1].options[op[1].selectedIndex].text;
+op[1].addEventListener('change', function(event) {
+    operationSign2 = op[1].options[op[1].selectedIndex].text;
+});
+ var operationSign3 = op[2].options[op[2].selectedIndex].text;
+ op[2].addEventListener('change', function(event) {
+    operationSign3 = op[2].options[op[2].selectedIndex].text;
+});
+   
+
+
+
+//Calculate 
+
+calculateButton.addEventListener('click', function() {    
+    resultContainer.innerHTML = '';
+    errorContainer.innerHTML = '';
+    
+    // (TO DO: A FUNCTION : DRY !!!)
+    if (operationSign1 == ":") {
+        operationSign1 = '/'
+    }
+    if (operationSign2 == ":") {
+        operationSign2 = '/'
+    }
+    if (operationSign3 == ":") {
+        operationSign3 = '/'
+    }
+    let sum = eval (`${parseInt(firstOperand.value)}  ${operationSign1}  ${parseInt(secondOperand.value)} ${operationSign2} ${parseInt(thirdOperand.value)} ${operationSign3} ${parseInt(fourthOperand.value)}`)
+    if (!sum) {
+       errorContainer.innerHTML = "Error!!! not a number"
+    } else {
+    resultContainer.innerHTML = sum;
+    }
+})
+
+
+// CHECK IF INPUT TYPE IS NUMBER 
+function checkIfNumber(operand) {
```

Comment:
in my opinion, it is better to do this by validating the numbers when you press the calculate button rather than every keypress because in this way I can copy paste and pass the validation or only the last key and it will be simpler.
But it is nice that you thought about this and read about keycode and ASCII ;)